### PR TITLE
Feat/rami/add GitHub mndy code temaplate app

### PR DESCRIPTION
**Pull request by rami-monday**

### CODE REVIEW

File: apps/github-monday-code/src/storage/base-secure-storage.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { SecureStorage } from '@mondaycom/apps-sdk';
```
Comment (DorShakedMonday) [inline apps/github-monday-code/src/storage/base-secure-storage.js:1]: Looks amazing mate 💪🏼

File: apps/github-monday-code/src/storage/base-storage.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+class BaseStorage {
+  constructor(token) {
+    this.storage = new Storage(token);
+  }
+  /**
+   * @typedef {Object} Options
+   * @property {string} [token] - The token value.
+   * @property {string} [previousVersion] - The previous version.
+   */
```
Comment (DorShakedMonday) [inline apps/github-monday-code/src/storage/base-storage.js:11]: I assume this is a mistake and should be deleted, correct? - These are the options for the set

**Review by DorShakedMonday — APPROVED** [review]
Approved with some small comments

File: apps/github-monday-code/src/storage/base-storage.js (hunk 2 above)
Comment (rami-monday) [reply apps/github-monday-code/src/storage/base-storage.js:11]: i see all operations have these options, in https://github.com/mondaycom/apps-sdk

Screen Shot 2023-07-19 at 11 46 31

**Pull request merged**